.github/workflows/run_comparative_benchmarks.yml

```
@@ -14,8 +14,56 @@ on:
     - cron: '0 09,21 * * *'
   workflow_dispatch:
 
+env:
+  GCS_DIR: gs://iree-samples-github-actions-${{ github.event_name == 'pull_request' && 'presubmit' || 'postsubmit' }}-artifacts/${{ github.run_id }}/${{ github.run_attempt }}
+
 jobs:
+  build_gpu:
+    runs-on:
+      - self-hosted  # must come first
+      - runner-group=presubmit
+      - environment=testing
+      - machine-type=c2-standard-16
```

We can use 96 cores build machine

```suggestion
      - cpu
```

---

Changed